### ISSUE TITLE
Action type none should be idle

### ISSUE THREAD

**Issue by EvoLandEco** [tresinformal/drakkar#656]
`none` could be misleading and less informative than `idle`, see the class `action_type`:

```c++
enum class action_type
{
  turn_left,
  turn_right,
  accelerate_forward,
  accelerate_backward,
  shoot,
  none,
  shoot_stun_rocket
};
```

suggested change:
```c++
enum class action_type
{
  turn_left,
  turn_right,
  accelerate_forward,
  accelerate_backward,
  shoot,
  none, >>>> idle,
  shoot_stun_rocket
};
```

**Comment by TheoPannetier** [comment]
Good idea! I think an easy way to implement this would be to simply change references to `action_type::none` in existing tests to `action_type::idle`, then work from there to make the tests pass :)

**Comment by EvoLandEco** [comment]
closed by #667
